core/capabilities/ccip/configs/solana/chain_writer.go

```diff
@@ -288,32 +274,36 @@ func GetSolanaChainWriterConfig(routerProgramAddress string, commonAddressesLook
 		return chainwriter.ChainWriterConfig{}, errors.New("from address cannot be empty")
 	}
 
-	// validate CCIP Router IDL, errors not expected
+	// validate CCIP Offramp IDL, errors not expected
```

Comment:
I saw the routerIDL is also used below, should we validate both ?

Reply:
Good point